helpers/dom.js

```diff
@@ -227,3 +227,19 @@ export function isVisible(node) {
 	return true;
 
 }
+
+export function querySelectorComposed(node, selector) {
+
+	const elem = node.querySelector(selector);
+	if (elem) return elem;
+
+	const allElems = node.querySelectorAll('*');
+	for (let i = 0; i < allElems.length; i++) {
```

Comment:
Nit: `i` isn't used here outside of indexing `allElems`, so you could instead use `for (const elem of allElems)`.

Reply:
Agreed -- fixed!